packages/pyright-internal/src/analyzer/typeDocStringUtils.ts

```diff
@@ -64,6 +64,7 @@ function isInheritedFromBuiltin(type: FunctionType | OverloadedFunctionType, cla
         type.details.moduleName === 'builtins' &&
         !!classType &&
         !!type.boundToType &&
+        !ClassType.isBuiltIn(type.boundToType) &&
         !ClassType.isSameGenericClass(classType, type.boundToType)
```

Comment:
If we take this change, I think the `isSameGenericClass` call is no longer needed. Presumably if `type` is a builtin, but `type.boundToType` is not, they are not the same class.

Reply:
I'm not familiar with this code. I think Jake wrote it. The change looks reasonable to me.